mds/locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,205 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
we already have `en_US` (this will let us define `en_GB` at some point if needed): a generic `en` translation is spurious.